src/content/plugins/split-chunks-plugin.md

```diff
@@ -294,6 +312,8 @@ module.exports = {
 };
 ```
 
+To see what info is available in `module` and `chunks` objects, you can put `debugger;` statement in the callback, [run your webpack build in debug mode](/contribute/debugging/#devtools) and inspect the parameters in Chromium DevTools.
```

Comment:
This one is very useful although i feel like this can sound much more like a documentation piece
```suggestion
In order to see what information is available in `module` and `chunks` objects, you can put `debugger;` statement in the callback. Then [run your webpack build in debug mode](/contribute/debugging/#devtools) to inspect the parameters in Chromium DevTools.
```